internal/pkg/cli/pipeline_delete.go

```diff
@@ -176,21 +179,34 @@ func (o *deletePipelineOpts) askAppName() error {
 }
 
 func (o *deletePipelineOpts) getSecret() error {
-	path, err := o.ws.PipelineManifestLegacyPath()
-	if err != nil {
-		return fmt.Errorf("get path to pipeline manifest: %w", err)
+	// Look for default secret name for GHv1 access token based on default pipeline name.
+	o.ghAccessTokenSecretName = o.pipelineSecretName()
+	output, err := o.secretsmanager.DescribeSecret(o.ghAccessTokenSecretName)
+	if err == nil {
+		// Check for Copilot-assigned tag for added assurance. If tags not found, fall through.
+		for _, tag := range output.Tags {
+			if aws.StringValue(tag.Key) == "copilot-application" && aws.StringValue(tag.Value) == output.CreatedDate.UTC().Format(time.UnixDate) {
+				log.Infof("Found secret '%s'.\n", o.ghAccessTokenSecretName)
```

Comment:
We might need to add some explanation of why we are looking for a secret to delete in these log info. People that are using non-GHV1 might find it super confusing and unsettling 😭

Reply:
The only people who would see this log info are GHv1 people, unless they just _happened_ to name a secret `github-token-appName-repoName` and also tagged it `copilot-application: [timestamp]`.

But yes, they may not have known/noticed that the secret was created for them when they created their pipeline, so I'll add some explanation for the GHv1 people! 😄